src/assets/_sass/_dds.scss

```diff
@@ -628,8 +641,21 @@ body {
 
 
 /* CSS below this line still needs to be organized and cleaned-up */
+
+// Mobile fix for replicating existing col offset with uswds 1.0 grid
+.usa-width-two-thirds.usa-offset-one-sixth{
+  @media (max-width: $screen-xs-min) {
+    margin-left: 0px;
```

Comment:
Small thing, but no need for units when the value is `0`.

Reply:
This one needs to be set at 0, it's overriding default margins on `.usa-offset-one-sixth` for mobile screens. The grid offset classes for the 1.0 standards aren't officially documented and not fully responsive.

Reply:
My comment wasn't super clear. `margin-left: 0;` should suffice here.

Reply:
A-ha 🤦‍♂️ ! Yes, of course.